submissions/israelOladosu/htmlForms/signUp.html

```diff
@@ -0,0 +1,100 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+ <meta charset="UTF-8">
+ <title>Sign Up Page</title>
+</head>
+
+<body>
+ <header>
+  <!-- container for the site logo -->
+  <div>
+   <a href="index.html">
+    <img src="./img/Farmville flat green.png" alt="jumia">
```

Comment:
Is this image your logo? If it is, I think you should update the name.
Also, update the name in all it has been used.